e2e/pages/index.ts

```diff
@@ -1 +1,2 @@
 export * from './allergies-page';
+export * from './patientBiometrics&VitalsPage';
```

Comment:
Use kebab case for file naming. @RandilaP

Reply:
Check all the file names

Reply:
@anjula-sack Done